source/help/apps/desktop-guide.rst

```diff
@@ -4,32 +4,32 @@ Using Mattermost Desktop App
 
 Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
 
-You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up.
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up and for troubleshooting tips.
 
 To view the latest updates, please see our `changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_.
 
-Below are a few tips to get you started and configuring your experience on the desktop app:
+Below are a few tips to get you started and to configure your experience on the desktop app:
 
  - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#id1>`_
  - `App Options <https://docs.mattermost.com/help/apps/desktop-guide.html#id2>`_
  - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#id3>`_
- - `Troubleshooting <https://docs.mattermost.com/help/apps/desktop-guide.html#id4>`_
 
 Team Management
 ---------------------------------------------------------------------
 
 You can connect to multiple Mattermost servers from a single interface on the desktop app.
 
-Team Management appears in the Settings page, available from the `File` menu under `Settings`.
+The Team Management section allows you to add, edit and remove servers and teams.
 
 Adding servers and teams
```

Comment:
Yeah, agree, it's a little misleading.

With the release of team sidebar feature, we can now easily display ALL teams in the same server tab, so we'll be renaming "team management" to "server management".

Previously you had to have a separate tab for each of your teams, if you wanted to quickly switch between them, or immediately see whether one of the teams has a mention or an unread. That's no longer the case as the team sidebar covers those.

I was planning to revisit the doc prior to the next Desktop App release (which is in February), and I'll make sure to address the point you raised here.